src/force/CabanaPD_ForceModels_PMB.hpp

```diff
@@ -168,16 +175,82 @@ struct ForceModel<PMB, Elastic, TemperatureDependent, TemperatureType>
     }
 };
 
-template <typename ModelType, typename DamageType, typename ThermalType,
-          typename ParticleType>
-auto createForceModel( ParticleType particles, const double delta,
+template <typename ParticleType>
+auto createForceModel( PMB, Elastic, ParticleType particles, const double delta,
                        const double K, const double alpha, const double temp0 )
 {
     auto temp = particles.sliceTemperature();
     using temp_type = decltype( temp );
-    return ForceModel<ModelType, DamageType, ThermalType, temp_type>(
+    return ForceModel<PMB, Elastic, TemperatureDependent, temp_type>(
         delta, K, temp, alpha, temp0 );
 }
+
+template <typename TemperatureType>
+struct ForceModel<PMB, Fracture, TemperatureDependent, TemperatureType>
+    : public ForceModel<PMB, Fracture, TemperatureIndependent>,
+      BaseTemperatureModel<TemperatureType>
+{
+    using base_type = ForceModel<PMB, Fracture, TemperatureIndependent>;
+    using base_temperature_type = BaseTemperatureModel<TemperatureType>;
+    using base_model = typename base_type::base_model;
+    using fracture_type = typename base_type::fracture_type;
+    using thermal_type = TemperatureDependent;
+
+    using base_type::c;
+    using base_type::delta;
+    using base_type::K;
+
+    // Does not use the base bond_break_coeff.
```

Comment:
Not sure this comment is clear.

Reply:
I assume you mean we need to compute the bond_break_coeff for thermo-mechanics as opposed to use the one pre-computed. However, in Line 238 we only use s0 for that. G0 is used in Line 228 though.

Perhaps something like "Parameters to compute temperature-dependent bond_break_coeff" or so? We could also add "(not using the base bond_break_coeff)".

Reply:
This comment was intended to explain why we don't use the base `bond_break_coeff`, but we do use the others